src/main/webapp/script.js

```diff
@@ -196,6 +205,8 @@ function populateAQVisualisationData() {
         if (printDebugData) {console.log(data);}
         loadHeatmap(aqData);
         scoreRoute(data);
+        updateCleanAirPlaces(data, mapBounds); // TODO (Rachel): disable clean air button until data is ready
```

Comment:
do you still need this TODO comment?

Reply:
Yep, I decided it wasn't something that I wanted to tackle in this PR, because I wanted to get something out for the base feature